docs/snippets/quickstart-install.md

```diff
@@ -1,22 +1,75 @@
+!!! todo "Installing the `quickstart` plugin"
+    === "Using a binary"
+         You can install the `quickstart` plugin by:
 
+         1. Download the binary for your system from the <a href="https://github.com/knative-sandbox/kn-plugin-quickstart/releases"> `quickstart` release page</a>.
 
-`konk` is a shell script that completes the following functions:
+         1. Rename the binary, make it executable, and rename it to `kn-quickstart`:
+
+             ```
+             mv kn-quickstart-darwin-amd64 kn-quickstart
+             chmod +x kn-quickstart
```

Comment:
attempting to follow the instructions on Mac, what happens is:

<img width="265" alt="image" src="https://user-images.githubusercontent.com/354013/126554299-98afafeb-550f-426a-a36d-a61dbc594af0.png">

To run it you have to go to system settings > security a> general and click "allow anyway" on a scary warning. TBH I don't think asking people to do this is realistically less scary than asking them to run `curl install.konk.dev` (which was the reason to do the plugin..), and it's four steps instead of 1. I think we need to figure out a less scary install process (maybe brew, for Mac?) before it makes sense to add this to the docs tbh :/

Reply:
For brew, we're waiting on https://github.com/knative-sandbox/homebrew-kn-plugins/pull/22

Reply:
good catch @julz  that you got the prompt.

It's weird that I didn't get the popup on my system when I tried today

Also, I ask how come with brew we will not have the popup? The brew method is not compiling the binary is downloading a binary so in theory it should also prompt. No?

Reply:
related: https://github.com/knative/client/issues/733 - looks like kn itself has same problem when downloaded as a binary directly